conf/log4j.properties

```diff
@@ -34,6 +37,14 @@ log4j.appender.MASTER_LOGGER.MaxBackupIndex=100
 log4j.appender.MASTER_LOGGER.layout=org.apache.log4j.PatternLayout
 log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%n
 
+# Remote appender for Master
+log4j.appender.REMOTE_MASTER_LOGGER=org.apache.log4j.net.SocketAppender
+log4j.appender.REMOTE_MASTER_LOGGER.Port=${alluxio.logserver.port}
+log4j.appender.REMOTE_MASTER_LOGGER.RemoteHost=${alluxio.master.hostname}
+log4j.appender.REMOTE_MASTER_LOGGER.ReconnectionDelay=10000
+log4j.appender.REMOTE_MASTER_LOGGER.filter.ID=alluxio.AlluxioRemoteLogFilter
+log4j.appender.REMOTE_MASTER_LOGGER.filter.ID.appenderName=MASTER_LOGGER
```

Comment:
maybe just call this `master` to have a more natural directory name

Reply:
Now `filter.ID.appenderName` has to be the same as the name of appender used by Alluxio servers to write their local logs, i.e. MASTER_LOGGER or WORKER_LOGGER.
We could use a different name, but then in `AlluxioLogServerProcess#configureHierarchy`, we need to perform more configuration in order to setup the logger used by the log server to write the remote logs to logserver's local storage.